src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestBase.cs

```diff
@@ -130,7 +130,6 @@ internal async Task RunClientServer(Func<QuicConnection, Task> clientFunction, F
                         await serverFunction(serverConnection);
                         serverFinished.Set();
                         clientFinished.Wait();
-                        await serverConnection.CloseAsync(0);
```

Comment:
Why are we removing close here?
cc @CarnaViire

Reply:
Because disposing the connection should be sufficient now.

Reply:
I would not remove it... While disposing *will* close the connection, it would be an abortive closing -- in my view it is mostly a precaution if a user forgot to call graceful CloseAsync. The `RunClientServer` scenario was intended to work gracefully.

If the problem is `ObjectDisposedException` in `Dispose_WithOpenLocalStream_LocalStreamFailsWithQuicOperationAbortedException` maybe it will be better to make closing a no-op if the connection was already disposed?

Reply:
Yeah, that does seem like a better approach. I'll try to make this work.